packages/nx-aws-cache/src/tasks-runner/runner.ts

```diff
@@ -1,5 +1,10 @@
 import { config as dotEnvConfig } from 'dotenv';
-dotEnvConfig();
+
+['.env', '.env.local', '.local.env'].forEach(file => {
```

Comment:
This should be just the opposite, because `.env.local` should always take precedence over `.env`. It should be the same as implemented here: https://github.com/nrwl/nx/blob/231a998e004c13e559c171ba6c164a9127886153/packages/nx/bin/nx.ts#L111-L115

The fact is, `dotEnvConfig` doesn't override a variable that is already set. To do that there's an option `dotEnvConfig({ path, override: true });`, but I would discourage to use this.

Reply:
The link to NX github I pasted before looks like an old version, which was fixed.

Reply:
https://github.com/nrwl/nx/pull/18746

Reply:
Thanks, I didnt see the updated implementation, that was what was confusing me. I'll adjust to mimic the new implementation.